src/styles/variables.scss

```diff
@@ -110,6 +110,10 @@ $icon-size--small: 12px;
 $icon-size--medium: 16px;
 $icon-size--large: 24px;
 
+// Selectors for applying clean-theme styling
+$annotator--theme-clean: '.annotator-frame--theme-clean';
+$sidebar--theme-clean: '.hypothesis-app--theme-clean';
```

Comment:
In the context of the sidebar we could use just `theme-clean` as the class name, since it isn't really a variant of the `hypothesis-app` component so much as a global style flag for the application.

In the context of the annotator there is a small risk of a conflict with a style on the page, but probably so small that we don't need to worry about it. Plus once we've switched the sidebar over to use Shadow DOM it won't be a problem any more.